back-end/www/models/model_operations/location_operations.py

```diff
@@ -0,0 +1,193 @@
+"""Functions to operate the location table."""
+
+from models.model import db
+from models.model import Location
+
+def create_location(factory_id, photo_year=1997, photo_url=""):
+    """
+    Create a location.
+
+    Parameters
+    ----------
+    factory_id : str
+        ID(uuid) provided by importing from the factory table
+    
```

Comment:
again no blank lines between the variables (as shown https://numpydoc.readthedocs.io/en/latest/format.html), please also fix similar problems in the code, thanks